invary_runtime_integrity/README.md

```diff
@@ -0,0 +1,44 @@
+# Invary Runtime Integrity
+
+Visualize the Runtime Integrity of your Invary managed operating systems.
+
+## Overview
+
+Invary validates the Runtime Integrity of operating systems and detects rootkits that can deceive other systems. This integration comes with an out-of-the-box dashboard that visualizes your Runtime Integrity, providing confidence in your runtime security or spotlighting endpoints that lack integrity and may be compromised. The Invary dashboard also provides insights into your operating system inventory at runtime, including distributions and kernel versions in use. 
```

Comment:
Can you add a bit more detail about exactly what kind of data will live on the dashboard? I see down further that you're submitting logs, it would be helpful to mention that here and talk about what kinds of logs are being submitted.

Reply:
Sure, I'll work on tweaks with our leadership team.

Reply:
@anawishnoff I have added some copy around trending and the appraisals our tool performs.